### PR TITLE
Custom training params

### PR DESCRIPTION
**Pull request by tmckeown311**
# Overview

**Type of Change:** Refactor

**Summary:** Added parameters to kfold and normal training programs that allow use of the hyperparameter tuning's results for better model training. Added parameters to the simulators as well.

## UI/UX Implementation

None

## Model Updates

None

### Data Models

None

### Object-Oriented Models

Simulators can now use custom parameters for loss rate, brightness, rotation, and batch size

## End-to-End Testing Instructions

Check that tests pass for the training program and run the simulators to ensure proper intended function.

### CODE REVIEW

File: src/training_program.py [hunk 1]
```diff
@@ -185,14 +185,14 @@ def get_train_test_split(self, df):
         image_binaries, labels, test_size=0.2, random_state=42)
         return [train_x, test_x, train_y, test_y]
 
-    def training_evaluation_resnet(self, num_epochs, train_loader, test_loader, view):
+    def training_evaluation_resnet(self, num_epochs, train_loader, test_loader, view, loss=0.001):
```
Comment (JoeCx) [inline src/training_program.py:188]: Lr doesnt stand for loss but stands for learning rate, rename variable to avoid confusion later.

**Review by JoeCx — APPROVED** [review]
Great job! Lrate make much more sense now, everything else looks great.
